src/oss.yml

```diff
@@ -24,6 +24,18 @@ paths:
     $ref: "./common/paths/authorizations.yml"
   /authorizations/{authID}:
     $ref: "./common/paths/authorizations_authID.yml"
+  /legacy/authorizations:
+    servers:
+      - url: "/private"
+    $ref: "./common/paths/authorizations.yml"
```

Comment:
I think to avoid the weird `$ref` values in the contracts, we'll need to stop reusing files between the v1 and v2 operations. I'd try:
1. Copy this file to `oss/paths/legacy_authorizations.yml`
2. Change the tag in the file from its current value (I assume `Authorizations`) to something like `Legacy Authorizations`
3. Update this ref to point at the new file

Then I'd do the same with the `authorizations_authID.yml` file below